compute-pcrs/Containerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/library/rust:1.89-trixie AS builder
+WORKDIR /compute-pcrs
+COPY Cargo.toml Cargo.lock .
+COPY compute-pcrs compute-pcrs
+# Hack: Set compute-pcrs as sole member to avoid needing to copy other crates.
+# In that case, a rebuild would be triggered upon any change in those crates.
+RUN sed -i 's/members =.*/members = ["compute-pcrs"]/' Cargo.toml && \
+    git clone --depth 1 https://github.com/confidential-clusters/reference-values && \
+    cargo build --release -p compute-pcrs
+
+FROM docker.io/library/debian:trixie
```

Comment:
can we use fedora as base image?

Reply:
Well, it can be built on the debian rust image, but you should copy it in a fedora base image. Or are you afraid that the dynamic library won't match?

Reply:
> Or are you afraid that the dynamic library won't match?

Yes I was, but maybe it's fine. I'll test.